supabase/migrations/39_alerts.sql

```diff
@@ -124,19 +124,23 @@ create or replace function internal.send_alerts()
 returns trigger as $trigger$
 begin
 
-perform
-  net.http_post(
-    url:='https://eyrcnmuzzyriypdajwdk.supabase.co/functions/v1/alert-data-processing',
-    headers:='{"Content-Type": "application/json", "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImV5cmNubXV6enlyaXlwZGFqd2RrIiwicm9sZSI6ImFub24iLCJpYXQiOjE2NDg3NTA1NzksImV4cCI6MTk2NDMyNjU3OX0.y1OyXD3-DYMz10eGxzo1eeamVMMUwIIeOoMryTRAoco"}'::jsonb,
-    body:=concat('{"time": "', now(), '"}')::jsonb
-  );
+if new.alert_type = 'data_not_processed_in_interval' then
+  perform
+    net.http_post(
+      'https://eyrcnmuzzyriypdajwdk.supabase.co/functions/v1/alert-data-processing',
+      to_jsonb(new.*),
+      headers:='{"Content-Type": "application/json", "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImV5cmNubXV6enlyaXlwZGFqd2RrIiwicm9sZSI6ImFub24iLCJpYXQiOjE2NDg3NTA1NzksImV4cCI6MTk2NDMyNjU3OX0.y1OyXD3-DYMz10eGxzo1eeamVMMUwIIeOoMryTRAoco"}'::jsonb
```

Comment:
Instead of deploying the edge function with the `--no-verify-jwt` flag, I set the `verify_jwt` property in the configuration file to `false` so that this setting is applied locally as well (ref: [Supabase CLI Configuration](https://supabase.com/docs/guides/cli/config#functions.function_name.verify_jwt)). Let me know if you see any issue with that.

Reply:
Even better